bin/cli.js

```diff
@@ -252,6 +253,10 @@ const SKIP_LOAD_CONFIG = {
   accounts: skipLoadConfigAccountSubCommands,
 };
 
+const SKIP_CONFIG_FLAG_VALIDATION = {
```

Comment:
Nit, but the naming of this is a little misleading. This looks like it's being used to skip the error that typically gets logged when the user passes in the `--config` flag while a global config exists.